docker-compose.yml

```diff
@@ -8,18 +8,18 @@ services:
       - .:/app  
     restart: unless-stopped
     depends_on:
+      - db
       - beanstalkd
     stdin_open: true
     tty: true
 
   db:
-    image: mysql:5.7
-    command: --default-authentication-plugin=mysql_native_password
+    image: postgres:13.0
     restart: always
     environment:
-      MYSQL_ROOT_PASSWORD: example
+      POSTGRES_PASSWORD: example
     ports:
-      - "3306:3306"
+      - "6543:5432"
```

Comment:
Forgot to ask, what's happening here as well?

Reply:
Just making sure `Postgres` ports doesn't conflict with already running Postgres database

Reply:
Cool, cool... I'm more used to adding a 1 at the end i.e. `54321` but I guess this works as well.